src/styles/theme.scss

```diff
@@ -1,7 +1,15 @@
-@use './core/utility.scss' as *;
+@use './tools/utility.scss' as *;
+@use './tools/flex.scss' as *;
 
 main[data-theme='light'],
```

Comment:
i might be able to show this more in a screen share but lilypad is not opinionated on how you go about implementing the dark and light them ( machine settins, toggle etc....). For example if you change your settings to dark theme mode lilypad doesn't care about that. the only thing controlling light or dark more is the "data-theme" value so if its dark then it will show dark etc... 

An end user (developer) needs to update that value how they see fit, Like for the turkey dash I actually have a dark light mode hook and theme provider I use to change "data-theme"
[
hook ](https://github.com/mozilla/turkey-portal/blob/main/client/hooks/useDarkMode.tsx)
[provider](https://github.com/mozilla/turkey-portal/blob/main/client/contexts/ThemeProvider.tsx)

hope that is what you are asking?!